tests/test_api.py

```diff
@@ -1344,3 +1344,36 @@ def test_get_list(self):
         self.assertIn('/horses', content)
         self.assertIn('/zebras', content)
         self.assertIn('/users', content)
+
+
+class TestCatsAPI(APITestCase):
+
+    """
+    Tests for nested resources
+    """
+
+    def setUp(self):
+        self.fixture = create_fixture()
+        home_id = self.fixture.locations[0].id
+        backup_home_id = self.fixture.locations[1].id
+        parent = Cat.objects.create(
+            name='Parent',
+            home_id=home_id,
+            backup_home_id=backup_home_id
+        )
+        self.kitten = Cat.objects.create(
+            name='Kitten',
+            home_id=home_id,
+            backup_home_id=backup_home_id,
+            parent=parent
+        )
+
+    def test_additional_sideloads(self):
```

Comment:
Can we add another test where we override that setting, and confirm that it gets used?